tests/NHSD.BuyingCatalogue.Organisations.Api.IntegrationTests/Features/Ods Organisations GetByOdsCode.feature

```diff
@@ -77,6 +77,7 @@ Scenario: If a user is not authorised then they cannot access the organisation
     When a GET request is made for an Ods organisation with code B1G
     Then a response with status code 401 is returned
 
+@ignore
```

Comment:
Can you add a brief comment explaining why this scenario is ignored?

Reply:
done